backend/twitterbot/top_contributors.py

```diff
@@ -0,0 +1,40 @@
+import collections
```

Comment:
I think this file should be in `backend/tournesol/utils/top_contributors.py`.

- This file has currently no "grip" with the twitter bot
- It is very generic, and could be used anywhere in the application
- Computing the top contributors should be the responsibility of the application providing the `Comparison` and `ContributorRating` model

Reply:
I agree, I changed it.